script/Arcueid/Day 10/10_00_ARC10_6.txt

```diff
@@ -451,12 +456,13 @@ Does this talk about immortality have anything to do with Arcueid's enemy?"
 [sha:64604d68684ef8031cbb9ef352bbd04bc0979f88]{
 -- Page 705, Offset 26167.
 -- とにかく死徒と区別される吸血鬼の『不老不死』は不安定なものなんだよ。ヤツらの場合、単純に寿命が人間の数倍になった、という事だけだからな」
-In any case, the immortality of a vampire that's distinct from a Dead Apostle is a precarious thing. In their case, they simply live several times longer than a human."
+The 'immortality' of a non-Dead Apostle vampire is an unstable thing. In their case, they simply live several times longer than humans."
 }
 [sha:362c0bda01e766d32ed6c5347dd2be4a3eabf29a]{
 -- Page 705, Offset 26168.
 -- 「……単純にって。あいつら、寿命だけじゃなくて力とかも人間離れしてるだろ」
-"...Is that all? It's not just their lifespan that's different, but their superhuman strength, right?"
+// VERY liberally TL'd
+"...Is that all? What about their superhuman strength?"
```

Comment:
I think this is fine since it's kinda implied, but probably revert just to be safe?
If revert I'd add "too" after "superhuman strength" though

Reply:
if you don't think anything's missing here, I'd rather keep it. The og line was pretty stilted